.circleci/config.yml

```
@@ -21,15 +30,21 @@ jobs:
       - run: ./build/build.sh
       - run: ./build/test.sh .
       - run: e2e-tests/test.sh
+
+  publish_dockerhub:
+    executor: docker-go
+    steps:
+      - setup_remote_docker
+      - checkout
+      - run: go install github.com/google/ko@latest
+      - run: echo "VERSION=$(cat version.txt | tr -d '\n')" >> $BASH_ENV
```

This should probably be … 

```suggestion
      - run: echo "VERSION=${CIRCLE_TAG#"v"}" >> $BASH_ENV
```

… to match the way we declare our release versions in other projects.

---

lets keep it as is for now, if it becomes an issue in the future we can revisit